test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeUnsafeTests.java

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2021 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package org.openj9.test.lworld;
+
+
+import jdk.internal.misc.Unsafe;
+import java.lang.reflect.Field;
+import static org.testng.Assert.*;
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeClass;
+import static org.openj9.test.lworld.ValueTypeUnsafeTestClasses.*;
+
+
+@Test(groups = { "level.sanity" })
+public class ValueTypeUnsafeTests {
+	static Unsafe myUnsafe = null;
+
+	@BeforeClass
+        static public void testSetUp() throws RuntimeException {
+                try {
+                        Field f = Unsafe.class.getDeclaredField("theUnsafe");
+                        f.setAccessible(true);
+                        myUnsafe = (Unsafe)f.get(null);
```

Comment:
The public method `getUnsafe()` in unsafe class is not working ? It can be used to return the unsafe instance.
https://github.com/eclipse-openj9/openj9/blob/c37e61a9d5caa7d4a946f911fb62810e59e84030/jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java#L1073

Reply:
Huh, it was not working when I was testing before but it's working now (maybe I was using sun.misc.Unsafe?). Regardless, I'll change it to getUnsafe() now